smbios_validation_tool/constants.py

```diff
@@ -72,3 +72,4 @@ class FieldValueRegexps(enum.Enum):
   SOCKET_DESIGNATION_REGEXP = r'(CPU|P)\d+'
   DEVICE_LOCATOR_REGEXP = r'(?:[^\d]*)\d+'
   BANK_LOCATOR_REGEXP = r'.*(Node|Channel).*'
+  VENDORS_REGEXP = r'(?i).*(Intel|Google|Dell|Lenova|Marvel|Nivida|AMD|Mellanox).*'
```

Comment:
I assume that's supposed to be "Lenovo", "Marvell", and "Nvidia"?  What should the value be for HPE?  Should we limit this field at all?  Or should we just allow "*"?

Reply:
I'm ok with `*`

Reply:
@seroyer thanks for the heads up on the typos. 
I am not sure what todo about HPE, there is some onus on the vendors to add the values for there equipment but i am unsure what they label there own stuff as.

Reply:
I'm ok with this, thanks.  I think we'll want to decide down the road if we want this to be exclusive to OPI qualified partners (not necessarily OPI members, those that have integration/release tests pass), or if we should just open it up to anything.